agent/Dockerfile

```
@@ -81,6 +81,19 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 
 RUN go test ./... -v && make -C pagebroker test
 
+# =============================================================================
+# Stage: PageBroker Builder
+# =============================================================================
+FROM go-base AS pagebroker-builder
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Hadolint (2.15.1)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)